### PR TITLE
foundationdb60: fix build

### PR DESCRIPTION
**Pull request by omasanori**
###### Motivation for this change

This fixes build failures on the `foundationdb60` package.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>foundationdb60</li>
    <li>python37Packages.foundationdb60</li>
    <li>python38Packages.foundationdb60</li>
    <li>python39Packages.foundationdb60</li>
  </ul>
</details>

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/servers/foundationdb/default.nix [hunk 1]
```diff
@@ -69,6 +69,7 @@ in with builtins; {
 
     patches = [
       ./patches/ldflags-6.0.patch
+      ./patches/misc-fixes-6.0.patch
```
Comment (SuperSandro2000) [inline pkgs/servers/foundationdb/default.nix:72]: That name is not describing.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Does upstream know about this problem? Is there a patch we could fetch instead?

File: pkgs/servers/foundationdb/default.nix (hunk 1 above)
Comment (omasanori) [reply pkgs/servers/foundationdb/default.nix:72]: Sure. I will revise it. Thank you for reviewing!

**Comment by omasanori** [comment]
> Does upstream know about this problem? Is there a patch we could fetch instead?

FoundationDB 6.0.x is no longer under active maintenance upstream, the latest commit on the branch was authored in 2019-03-12.

File: pkgs/servers/foundationdb/default.nix (hunk 1 above)
Comment (omasanori) [reply pkgs/servers/foundationdb/default.nix]: Renamed to `include-fixes-6.0.patch` and added some description into the commit message. Is it good enough?

**Pull request merged**